flexget/components/sites/sites/iptorrents.py

```diff
@@ -135,9 +136,14 @@ def search(self, task, entry, config=None):
             query = normalize_unicode(search_string)
             query = quote_plus(query.encode('utf8'))
 
-            url = "{base_url}/t?{filter}&q={query}&qf=".format(
-                base_url=BASE_URL, filter=filter_url, query=query
-            )
+            if config.get('free'):
```

Comment:
i know it's not your code but we might as well improve it. All query params should be passed inside a params dict and not added to the url string like that.
Also, please use fstrings instead of `.format()`. 

If it's possible to change these formats to fstring in other places in the plugin that would be swell

Reply:
I believe I've addressed these improvements, please let me know if there's anything further that I can do.